src/border_agent/border_agent.hpp

```diff
@@ -126,6 +126,14 @@ class BorderAgent : private NonCopyable
      */
     void SetEnabled(bool aIsEnabled);
 
+    /**
+     * This method enables/disables the Border Agent ePSKc feature.
+     *
+     * @param[in] aIsEnabled  Whether to enable the BA ePSKc feature.
+     *
+     */
+    void SetEpskcEnabled(bool aIsEnabled);
```

Comment:
nit: may be consistent with the ot API signature as https://github.com/openthread/openthread/blob/20c9bcbc4142fd74e490bd50faa7def9e1b3a823/include/openthread/border_agent.h#L208

```suggestion
    void SetEphemeralKeyEnabled(bool aIsEnabled);
```

Reply:
Accept. Thanks~